### PR TITLE
RSEM easyconfigs

### PR DESCRIPTION
**Pull request by pescobar**
@boegel here you have another two easyconfigs using the glob support in makecp block

### CODE REVIEW

**Comment by fgeorgatos** [comment]
Test report by @fgeorgatos
**FAILED**
Build succeeded for 2 out of 5
Linux debian 6.0.10, Intel(R) Xeon(R) CPU           L5640  @ 2.27GHz, Python 2.6.6
See https://gist.github.com/85371a2b1f73c9ab7c56 for a full test report.

File: easybuild/easyconfigs/r/RSEM/RSEM-1.2.12-goolf-1.4.10.eb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+parallel = 1
+
+# put here the list of generated executables when compiling
+list_of_executables = ["extract-transcript-to-gene-map-from-trinity", "convert-sam-for-rsem",
+                       "rsem-*"]
+
+# this is the real EasyBuild line to copy all the executables and perl scripts to "bin"
+# glob support in MakeCp easyblock is required so EasyBuild-1.12 or higher is required
+easybuild_version = '1.12.0'
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/r/RSEM/RSEM-1.2.12-goolf-1.4.10.eb]: I'd remove L40-L43, redundant by today

**Comment by fgeorgatos** [comment]
this looks good, yet requires more testing...

File: easybuild/easyconfigs/r/R/R-3.0.2-goolf-1.4.10-bare.eb [hunk 2]
```diff
@@ -0,0 +1,37 @@
+name = 'R'
+version = "3.0.2"
+versionsuffix = '-bare'  # bare, as in no extensions included
+
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/r/R/R-3.0.2-goolf-1.4.10-bare.eb]: minor changes needed here, `easyblock = 'ConfigureMake'` , put this in REVIEW plz!
(not sure if getting in this historic R ec is useful any more)

**Comment by verdurin** [comment]
I suggest this PR be closed, given that `RSEM` is available now, albeit for different toolchains than listed here.

Unless these specific older versions are required?

**Comment by pescobar** [comment]
closing as this is already provided in easybuild

**Pull request closed without merging**